### PR TITLE
Changes based on pull request comments

### PR DESCRIPTION
**Pull request by gmellemstrand**
- Added tests
- Changed parameter
- Fixed bug when linking to edges generated by WalkableAreaBuilder

### CODE REVIEW

**Comment by gmellemstrand** [comment]
Må også legge til i build-config

extraEdgesStopPlatformLink: true

**Review by seime — COMMENTED** [review]
Mix of 2 different PRs ?

**Comment by gmellemstrand** [comment]
Basert på kommentarer fra OTP hovedfork pull request + bug fiks

File: src/main/java/org/opentripplanner/common/LuceneIndex.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ private void index() {
             //directory = new RAMDirectory(); // only a little faster
             IndexWriterConfig config = new IndexWriterConfig(Version.LUCENE_47, analyzer).setOpenMode(OpenMode.CREATE);
             final IndexWriter writer = new IndexWriter(directory, config);
-            for (Stop stop : graphIndex.stopForId.values()) {
+            for (Stop stop : graphIndex.stationForId.values()) {
```
Comment (seime) [inline src/main/java/org/opentripplanner/common/LuceneIndex.java]: Belongs to a different internal PR

**Comment by gmellemstrand** [comment]
LuceneIndex changes removed

**Pull request merged**